Compilation error when using REQ_LEFT

I'm running into problems using REQ_LEFT.

The following code doesn't compile:

```
package main

import (
	gc "github.com/rthornton128/goncurses"
)

func main() {
	menuWin, _ := gc.Init()

	gc.Raw(true)
	gc.Echo(false)
	menuWin.Keypad(true)
	gc.Cursor(0)

	menuWin.Keypad(true)
	itemPos, _ := gc.NewItem("YES", "")
	itemNeg, _ := gc.NewItem("NO","")

	choices := []*gc.MenuItem {itemNeg, itemPos}
	menu, _ := gc.NewMenu(choices)
	menu.SetWindow(menuWin)
	height, width := dimensions(menuWin)
	menu.SubWindow(menuWin.Derived(height, width, 0, 0))

	menu.Format(1, 2)
	menu.Post()

	menuWin.Refresh()

	var key gc.Key
	for ; ; key = menuWin.GetChar(){
		switch key {
		case gc.KEY_RIGHT:
			menu.Driver(gc.REQ_RIGHT) // Works
		case gc.KEY_LEFT:
			menu.Driver(gc.REQ_LEFT) // Compilation error
		}
	}

	gc.End()
}

func dimensions(w *gc.Window) (height int, width int) {
	y, x := w.YX()
	maxy, maxx := w.MaxYX()
	height = maxy - y
	width = maxx - x
	return
}

```
Compilation error:
```
$ go build req_left.go 
# command-line-arguments
./req_left.go:36:15: cannot use goncurses.REQ_LEFT (type goncurses.MenuDriverReq) as type int in argument to menu.Driver
```

However if I change REQ_LEFT to REQ_RIGHT it does compile.

```
$ go version
go version go1.13.8 linux/amd64
```
goncurses version seems to be `v0.0.0-20200419155752-a72cfe131f63` according to my `go.mod` file.

--- a/ext_defs.go
+++ b/ext_defs.go
@@ -91,22 +91,22 @@ type MenuDriverReq C.int
 
 const (
 	REQ_LEFT          MenuDriverReq = C.REQ_LEFT_ITEM
-	REQ_RIGHT                       = C.REQ_RIGHT_ITEM
-	REQ_UP                          = C.REQ_UP_ITEM
-	REQ_DOWN                        = C.REQ_DOWN_ITEM
-	REQ_ULINE                       = C.REQ_SCR_ULINE
-	REQ_DLINE                       = C.REQ_SCR_DLINE
-	REQ_PAGE_DOWN                   = C.REQ_SCR_DPAGE
-	REQ_PAGE_UP                     = C.REQ_SCR_UPAGE
-	REQ_FIRST                       = C.REQ_FIRST_ITEM
-	REQ_LAST                        = C.REQ_LAST_ITEM
-	REQ_NEXT                        = C.REQ_NEXT_ITEM
-	REQ_PREV                        = C.REQ_PREV_ITEM
-	REQ_TOGGLE                      = C.REQ_TOGGLE_ITEM
-	REQ_CLEAR_PATTERN               = C.REQ_CLEAR_PATTERN
-	REQ_BACK_PATTERN                = C.REQ_BACK_PATTERN
-	REQ_NEXT_MATCH                  = C.REQ_NEXT_MATCH
-	REQ_PREV_MATCH                  = C.REQ_PREV_MATCH
+	REQ_RIGHT         MenuDriverReq = C.REQ_RIGHT_ITEM
+	REQ_UP            MenuDriverReq = C.REQ_UP_ITEM
+	REQ_DOWN          MenuDriverReq = C.REQ_DOWN_ITEM
+	REQ_ULINE         MenuDriverReq = C.REQ_SCR_ULINE
+	REQ_DLINE         MenuDriverReq = C.REQ_SCR_DLINE
+	REQ_PAGE_DOWN     MenuDriverReq = C.REQ_SCR_DPAGE
+	REQ_PAGE_UP       MenuDriverReq = C.REQ_SCR_UPAGE
+	REQ_FIRST         MenuDriverReq = C.REQ_FIRST_ITEM
+	REQ_LAST          MenuDriverReq = C.REQ_LAST_ITEM
+	REQ_NEXT          MenuDriverReq = C.REQ_NEXT_ITEM
+	REQ_PREV          MenuDriverReq = C.REQ_PREV_ITEM
+	REQ_TOGGLE        MenuDriverReq = C.REQ_TOGGLE_ITEM
+	REQ_CLEAR_PATTERN MenuDriverReq = C.REQ_CLEAR_PATTERN
+	REQ_BACK_PATTERN  MenuDriverReq = C.REQ_BACK_PATTERN
+	REQ_NEXT_MATCH    MenuDriverReq = C.REQ_NEXT_MATCH
+	REQ_PREV_MATCH    MenuDriverReq = C.REQ_PREV_MATCH
 )
 
 // Menu Options

--- a/menu.go
+++ b/menu.go
@@ -80,7 +80,7 @@ func (m *Menu) Current(mi *MenuItem) *MenuItem {
 
 // Driver controls how the menu is activated. Action usually corresponds
 // to the string returned by the Key() function in goncurses.
-func (m *Menu) Driver(daction int) error {
+func (m *Menu) Driver(daction MenuDriverReq) error {
 	err := C.menu_driver(m.menu, C.int(daction))
 	return ncursesError(syscall.Errno(err))
 }

